PROPOSAL.md

```diff
@@ -3,6 +3,22 @@
 ## General Idea
 The idea of this project is to set up a kubernetes cluster and observe the cluster with Dynatrace. The kubernetes cluster runs one node and multiple pods and is hosted on Google Kubernetes Engine (GKE). Several microservices - communicating between each other - are running on the K8s cluster. Dynatrace is used to monitor traces between microservices, logs, metrics and the health of the K8s cluster.
 
+### Focus
+As our topic allows for various different points of focus, we explicitly outline what we will focus during our presentation:
+- Communication between the microservices
+  - Dynatrace PurePaths
+- Logs the microservices produce
+
```

Comment:
I think we can also add K8s monitoring, because Dynatrace also provides some insight for that